src/dswx_sar/detect_inundated_vegetation.py

```diff
@@ -100,10 +101,28 @@ def run(cfg):
             np.squeeze(rtc_dual[copol_ind, :, :]),
             np.squeeze(rtc_dual[crosspol_ind, :, :]))
 
-        filt_ratio = filter_SAR.lee_enhanced_filter(
-            rtc_ratio,
-            win_size=filter_size)
-
+        if filter_method == 'lee':
+            filtering_method = filter_SAR.lee_enhanced_filter
+            filter_option = {'win_size':
```

Comment:
maybe it would be convenient to use `vars()` to convert `SimpleNameSpace` into `Dict`. For example:

```python
filter_option = vars(filter_options.lee_filter)
```
One caveat is that we make sure that the member name in the `SimpleNameSpace` is the same as the parameter name of the filter functions.